dolphinscheduler-master/src/main/resources/application.yaml

```diff
@@ -64,10 +64,10 @@ spring:
       org.quartz.threadPool.makeThreadsDaemons: true
       org.quartz.threadPool.threadCount: 25
       org.quartz.jobStore.misfireThreshold: 60000
-      org.quartz.scheduler.batchTriggerAcquisitionMaxCount: 1
       org.quartz.scheduler.makeSchedulerThreadDaemon: true
       org.quartz.jobStore.driverDelegateClass: org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
       org.quartz.jobStore.clusterCheckinInterval: 5000
+      org.quartz.scheduler.batchTriggerAcquisitionMaxCount: 20
```

Comment:
And I don't think we need to change the default value of `org.quartz.scheduler.batchTriggerAcquisitionMaxCount`, this is the default value of quartz, and 20 is not the best one, the user still need to change if they need to have a higher output.

Reply:
can be 1 as well but need to add it

Reply:
We already have `org.quartz.scheduler.batchTriggerAcquisitionMaxCount: 1` at this yaml file.

Reply:
OK